example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -56,6 +56,9 @@ server:
     tls:
       serverCertPath: dev/tls/gardener-controller-manager.crt
       serverKeyPath: dev/tls/gardener-controller-manager.key
+shootBackup:
+  schedule: "0 * * * *"
```

Comment:
@swapnilgm, @rfranzke Can we use this schedule in the example or should we rather use `0 */24 * * *`?

Reply:
Use 0 */24 * * *., as examples are supposed to have recommended/default configuration.